spec/lib/process_settings/target_spec.rb

```diff
@@ -255,5 +255,83 @@
 
       expect(result.json_doc).to eq(false)
     end
+
+    describe "when target hash has nested values" do
+      subject { described_class.new(target_hash).with_static_context(context_hash).json_doc }
+
+      describe "when the context contains the nested structure of the target hash" do
+        let(:target_hash) do
+          {
+            'services' => {
+              'region' => 'west'
+            }
+          }
+        end
+
+        describe "when the target matches the context" do
+          let(:context_hash) do
+            {
+              'services' => {
+                'region' => 'west'
+              }
+            }
+          end
+
+          it { should eq(true) }
```

Comment:
I've gotta start using this syntax myself! It's so odd, it's hard to remember. I tried searching for it a few days ago and it was hard to find!
I wonder if you might want to send a PSA about `subject` and `it { should ...}`? It's awesome, yet I don't see anyone else using it.

Reply:
It's mentioned in the new test writing best practices. But I can definitely throw together a quick RSpec 101 lightning talk to run people through.

Reply:
A lightning talk sounds even better!